### DISCUSSION TITLE
Runout sensor pinout missing

### DISCUSSION THREAD

**Discussion by kdobrev** [bassamanator/everything-sovol-sv06#25] [Q&A]
The detailed pinout connection for a runout sensor is missing in the explanation. Whts is the logic for a filament runout on the SV06? 

The connector has 3 pins - V, G and S. What is the logic? Does S goes to G or it goes to V when a runout is detected?

![20230611_174131](https://github.com/bassamanator/everything-sovol-sv06/assets/8297386/fe1bbe85-ce9e-4c5b-a9d9-7e45ce3c892f)
![Uploading probe.png…]()
![Uploading fil-sensor-side.jpg…]()

**Comment by bassamanator** [comment]
>In order to get the filament sensor working, just make sure that the VCC, Ground, and Signal line up with the pins on the hotend PCB, port P7.

What's confusing about this?

**Comment by albertoabrao** [comment]
I assume he wants to know which is which. The picture should be enough, though.

Now,  should mention is that I have two runout sensors that look exactly the same. Yet, one of them requires me to invert pin 1 and 3. Else, there's clearly some kind of short on the board.

After that, the only thing is verifying if the probe is NC (normally closed) or NO (normally open), and adjusting the firmware as necessary. I am not familiar with how this is done on Klipper, so I can't help there.
